Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/MarkdownTextBlock.cs

```diff
@@ -50,6 +51,12 @@ public sealed class MarkdownTextBlock : Control, ILinkRegister
         public event EventHandler<LinkClickedEventArgs> LinkClicked;
 
         /// <summary>
+        /// Fired when an image from the markdown document needs to be resolved.
+        /// The default implementation is basically <code>new BitmapImage(new Uri(e.Url));</code>.
+        /// </summary>
+        public event EventHandler<ResolveImageEventArgs> ResolveImage;
```

Comment:
Yeah, you're right. Like I said the name is open for discussion.  
**ImageRequested** sounds fine to me. I will change it.